src/ZuluSCSI_log_trace.cpp

```diff
@@ -100,6 +100,8 @@ static const char *getCommandName(uint8_t cmd)
         case 0xA8: return "Read12";
         case 0xC0: return "OMTI-5204 DefineFlexibleDiskFormat";
         case 0xC2: return "OMTI-5204 AssignDiskParameters";
+        case 0xE0: return "RAM Diagnostic";
+        case 0xE4: return "Drive Diagnostic";    
```

Comment:
@peclark1 
I think prefixing these string with `Xebec ` would make them more inline with the two Xebec commands listed above.
https://github.com/ZuluSCSI/ZuluSCSI-firmware/blob/8671a0b6c40437300d59e0e5f4ed34b5547eb14a/src/ZuluSCSI_log_trace.cpp#L52-L53

`"Xebec RAM Diagnostic"`
`"Xebec Drive Diagnostic"`